src/open_inwoner/templates/pages/category/detail.html

```diff
@@ -35,7 +37,7 @@ <h1 class="h1">
     {% if questionnaire_roots %}
         <div class="grid">
             <div class="column column--start-1 column--span-6 ">
-                <aside class="questionnaire">
+                <aside class="questionnaire" aria-label="{% trans "zelfdiagnose" %}">
```

Comment:
I'm not sure whether these labels add value since the assistive technology probably can read the textual content.

 > Note: aria-label is intended for use on interactive elements, or elements made to be interactive via other ARIA declarations, when there is no appropriate text visible in the DOM that could be referenced as a label

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Reply:
Removed the aria-label here